twake/backend/node/src/services/workspaces/web/routes.ts

```diff
@@ -89,10 +89,10 @@ const routes: FastifyPluginCallback<{
       if (!workspace) {
         throw fastify.httpErrors.notFound(`Workspace ${workspaceId} not found`);
       } else {
-        throw fastify.httpErrors.forbidden("Only workspace admin can perform this action");
+        throw fastify.httpErrors.forbidden("You must be member of the workspace");
       }
-    } else if (workspaceUser.role !== "admin") {
-      throw fastify.httpErrors.forbidden("Only workspace admin can perform this action");
+    } else if (workspaceUser.role !== "moderator") {
```

Comment:
Could we replace this by the utils function "hasWorkspaceAdminLevel()" ?

Reply:
done